include/execution.hpp

```diff
@@ -172,6 +172,14 @@ namespace std::execution {
       sender<S> &&
       __has_sender_types<sender_traits<remove_cvref_t<S>>>;
 
+  template<class S, class... Ts>
+    concept sender_of =
+      typed_sender<S> &&
+      same_as<
+        __types<Ts...>,
+        typename sender_traits<S>::template
+            value_types<__types, type_identity_t>>;
+
```

Comment:
This comes from a different commit on main, I think.

Reply:
Yah not sure how this got in here?

Reply:
Fixed.